cognite/neat/core/rules/exporter/rules2triples.py

```diff
@@ -0,0 +1,20 @@
+from cognite.neat.core.rules.models import TransformationRules
+
+
+def get_instances_as_triples(transformation_rules: TransformationRules) -> list[tuple]:
```

Comment:
I think we should introduce a type for Triples in `neat`. I added one in `utils.utils` line 17. But that I think is different from this one, but then we should support both.  `tuple` is too generic